### ISSUE TITLE
Sanitizer in Validator middleware does not work

### ISSUE THREAD

**Issue by Toanzzz** [honojs/hono#660]
I tried to make some params always have lowercase, and figure that using the `addSanitizer` func in the validation pipe would be the most intuitive, except that it doesn't work at all 😅 (tried the `trim` function, but it also does nothing)

You can check the code below to reproduce it:

### Reproduce code:

```ts
import { Hono } from 'https://deno.land/x/hono@v2.5.0/mod.ts'
import { serve } from 'https://deno.land/std@0.163.0/http/server.ts'
import { validator } from 'https://deno.land/x/hono@v2.5.0/middleware.ts'

export const lowCase = (s: unknown) => `${s}`.toLowerCase()

const app = new Hono()
app.post(
  '/json',
  validator(v => ({
    trim: v.json('trim').trim().isOptional(),
    low: v.json('low').addSanitizer(lowCase).isOptional(),
  })),
  ctx => ctx.json(ctx.req.valid())
)

await serve(app.fetch)
```

### Sample request (using [Httpie](https://httpie.io/))

```sh
http POST localhost:8000/json 'trim==    b    ' low==CDF --raw '{
  "trim": "   a   ",
  "low":"ABC"
}'
```

### Expected result

```json
{
  "trim": "a",
  "low": "abc"
}
```

### Actual result 🚫

```json
{
  "trim": "   a   ",
  "low": "ABC"
}
```

**Comment by yusukebe** [comment]
Hi @Toanzzz ! 

Thank you for creating the issue.

Currently, the result values are not sanitized as you said. Some codes have to implement are lacking. I'll try to write them. Please wait.

**Comment by yusukebe** [comment]
Hi @Toanzzz !

This is fixed with #661 by @yujong-lee , we'll release the new version including this fix soon. Thank you!

**Comment by yusukebe** [comment]
It's released! I'll close this issue.
